test/golden/create-accounts-expected.txt

```diff
@@ -1,4 +1,4 @@
-- output: HLIEBEUgf32fElyvoriTOISAKAdnV0EGOp6xnyHRb38
+- output: npos8cElfalyz9qQL-nVwTaWKXUYlSk_QX7X4CujGwE
```

Comment:
This is expected. Before the coin base upgrade check for test versions was based on `_blockHeight parentHeader == 0`. Since the test mines a block at height 11, the upgrade was not applied. 

Now it depends on the the block height of the current header as stored in `PublicData`, namely `_pdBlockHeight`, to be 1. The value of `_pdBlockHeight` is incremented by pact service on execution of each block. For tests that skip ahead in the history, e.g. by using `someTestVersionHeader`, this value doesn't match the actual height of the header. For this test the value is `1` and the upgrade is applied during the test.

The synchronization of public data and the actual data for the current chain context is an issue that should be resolved in a separate PR.

Reply:
We should probably move these to more than just a hash anyway as they are of questionable value as is.